src/y-monaco.js

```diff
@@ -53,14 +57,26 @@ const createMonacoSelectionFromRelativeSelection = (editor, type, relSel, doc) =
   return null
 }
 
+/** 
+ * @type {ClassNameGetter}
+ */
+const defaultGetClassName = (state, clientID, isHead) => {
+  if (isHead) {
+    return 'yRemoteSelectionHead yRemoteSelectionHead-' + clientID
+  } else {
+    return 'yRemoteSelection yRemoteSelection-' + clientID
+  }
+}
+
 export class MonacoBinding {
   /**
    * @param {Y.Text} ytext
    * @param {monaco.editor.ITextModel} monacoModel
    * @param {Set<monaco.editor.IStandaloneCodeEditor>} [editors]
    * @param {Awareness?} [awareness]
+   * @param {ClassNameGetter?} [getClassName]
```

Comment:
This results in a parameter type of `getClassName: ClassNameGetter | null | undefined`, so we get "Object is possibly null" errors when it is invoked.

Dropping the `?` produces the correct parameter type `getClassName: ClassNameGetter | undefined`.

```suggestion
   * @param {ClassNameGetter} [getClassName]
```